src/main/java/uk/gov/hmcts/dm/utils/StringUtils.java

```diff
@@ -9,4 +13,15 @@ public static final String sanitiseFileName(String originalDocumentName) {
         return originalDocumentName == null ? null :
                 originalDocumentName.replaceAll("[^a-zA-Z0-9\\.\\-_+ ]","");
     }
+
+    public static String convertValidLogString(String log) {
+        List<String> list = Arrays.asList("%0d", "\r", "%0a", "\n");
+
+        // normalize the log content
+        String encode = Normalizer.normalize(log, Normalizer.Form.NFKC);
+        for (String toReplaceStr : list) {
+            encode = encode.replace(toReplaceStr, "");
```

Comment:
to me, chaining the replace calls would be more readable, but it's just an opinion
encode.replace("%0d", "")
            .replace("\r", "")
            .replace("%0a", "")
            .replace("\n", "")